EventPlaning.BL.Tests/EventManagerTests.cs

```diff
@@ -294,14 +346,14 @@ public void PatchEventFailEventNotExist()
                 Title = "BG and VG event",
                 DateFrom = "2018-01-23",
                 DateTo = "2018-01-23",
-                Address = new string[]
+                Address = new Address()
                 {
-                    "new g. 1",
-                    null,
-                    "Vilniaus m.",
-                    "Vilnius",
-                    "Lithuania",
-                    "LT-08303"
+                    AddressLine1 = "new g.1",
+                    City = "Vilnius",
+                    Country = "Lithuania",
+                    Province = "Vilniaus m.",
+                    EventId = 1,
+                    PostalCode = "LT-08303",
```

Comment:
Postal codes most of the time are numeric. LT has also ONLY numeric postal codes (the LT part is to indicate that it is Lithuania, however, Country part also states that.
Nothing to do here, just a note 🚀 👨‍🚀